demos/common/cpp/utils/include/utils/common.hpp

```diff
@@ -322,8 +323,9 @@ void logBasicModelInfo(const std::shared_ptr<ov::Model>& model) {
         const std::string name = output.get_any_name();
         const ov::element::Type type = output.get_element_type();
         const ov::PartialShape shape = output.get_partial_shape();
+        const ov::Layout layout = ov::layout::get_layout(output);
 
-        slog::info << name << ", " << type << ", " << shape << slog::endl;
+        slog::info << name << ", " << type << ", " << shape << ", " << layout.to_string() << slog::endl;
```

Comment:
It's already on master, why github don't see it, strange
https://github.com/openvinotoolkit/open_model_zoo/blob/5167a7715ea7b65e07823d143fc19a019c565e9a/demos/common/cpp/utils/include/utils/common.hpp#L326

Reply:
not a problem anyway